### ISSUE TITLE
Validation & testing result doesnt match

### ISSUE THREAD

**Issue by tapu1996** [yanx27/Pointnet_Pointnet2_pytorch#271]
In a task, I am using exact same validation set as testing. During testing, I am using saved weights. However, every time I run, it provides different logit as output which result in performance mismatch between validation and test. What could be the reason?
I have made sure that no augmentation is done during validation/testing. Pointcloud is exactly same as well.

**Comment by GH-Fonic** [comment]
Hi，bro.Did you solve the problem? I had the same problem.

**Comment by GH-Fonic** [comment]
I found the cause of the problem was ScannetDatasetWholeScene class， ,I used class S3DISDataset for prediction and it was worked.
